modos/cli.py

```diff
@@ -143,20 +151,18 @@ def create(
     endpoint = ctx.obj.endpoint
     # Initialize object's directory
     if endpoint:
-        s3 = list_endpoints(endpoint)["s3"]  # type: ignore
+        s3 = EndpointManager(endpoint).s3
         fs = connect_s3(s3, {"anon": True})  # type: ignore
-        if fs.exists(object_directory):
-            raise ValueError(
-                f"Remote directory already exists: {object_directory}"
-            )
-    elif object_directory.exists():
-        raise ValueError(f"Directory already exists: {object_directory}")
+        if fs.exists(object_path):
+            raise ValueError(f"Remote directory already exists: {object_path}")
+    elif object_path.exists():
```

Comment:
```suggestion
    elif Path(object_path).exists():
```
Only needed, if `OBJECT_PATH_ARG` is parsed as string